similarity/t/test_split_song.py

```diff
@@ -2,12 +2,11 @@
 
 from similarity.split_song import split_song
 
+from filehandler.handle_audio import get_MonoLoaded_Song
 
-def test_split_song():
-    dirname = os.path.dirname(__file__)
-    test_filename = os.path.join(
-        dirname, "test_split_song/8376-1-1_Demolition_Man_proud_music_preview.wav")
 
-    segments = split_song(test_filename)
+def test_split_song():
+    song = get_MonoLoaded_Song("similarity/t/test_split_song/8376-1-1_Demolition_Man_proud_music_preview.wav")
```

Comment:
106 chars in a line, whilst only 79 is permitted according to PEP8.

https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
:+1: to Casper's comment.